File: viajes/src/gestorAplicacion/adminVuelos/Pasajero.java
package gestorAplicacion.adminVuelos;
import java.io.Serializable;
public class Pasajero implements Serializable {
	
	private static final long serialVersionUID = 1L;
	private String pasaporte;
	public String nombre;
	private Tiquete tiquete;
	private int edad;
	private String email;

	public Pasajero(String pasaporte, String nombre, Tiquete tiquete, int edad, String email) {
		this.pasaporte = pasaporte;
		this.nombre = nombre;
		this.tiquete = tiquete;
		this.edad = edad;
		this.email = email;
		tiquete.setPasajero(this);
	}

	//GETTERS Y SETTERS

	public String getPasaporte() {
		return pasaporte;
	}
	public void setPasaporte(String pasaporte) {
		this.pasaporte = pasaporte;
	}
	public Tiquete getTiquete() {
		return tiquete;
	}
	public void setTiquete(Tiquete tiquete) {
		this.tiquete = tiquete;
	}
	public int getEdad() {
		return edad;
	}
	public void setEdad(int edad) {
		this.edad = edad;
	}
	public String getEmail() {
		return email;
	}
	public void setEmail(String email) {
		this.email = email;
	}



}

File: carpeta/gestorAplicacion/adminVuelos/Pasajero.py
class Pasajero:

    def __init__(self, pasaporte, nombre, tiquete, edad, email):
        self._pasaporte = pasaporte
        self.nombre = nombre
        self._tiquete = tiquete
        self._edad = edad
        self._email = email
        tiquete.setPasajero(self)

    #GETTERS Y SETTERS

    def getPasaporte(self):
        return self._pasaporte

    def setPasaporte(self, pasaporte):
        self._pasaporte = pasaporte

    def getTiquete(self):
        return self._tiquete

    def setTiquete(self, tiquete):
        self._tiquete = tiquete

    def getEdad(self):
        return self._edad

    def setEdad(self, edad):
        self._edad = edad

    def getEmail(self):
        return self._email
        
    def setEmail(self, email):
        self._email = email


